cg/meta/workflow/prepare_fastq.py

```diff
@@ -0,0 +1,88 @@
+"""API to prepare fastq files for analysis"""
+
+import logging
+import os
+from typing import Dict
+from pathlib import Path
+
+from cg.meta.compress.compress import CompressAPI
+from cg.apps.crunchy import CrunchyAPI
+from cg.meta.compress import files
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.store import Store
+from housekeeper.store import models as hk_models
+
+LOG = logging.getLogger(__name__)
+
+
+class PrepareFastqAPI:
+    def __init__(self, store: Store, compress_api: CompressAPI):
+        self.store = store
```

Comment:
Why not here as well?
```suggestion
        self.store: Store = store
```